packages/plugin-core/src/web/engine/store/VSCodeFileStore.ts

```diff
@@ -161,3 +159,20 @@ export async function getAllFilesWithTypes(opts: GetAllFilesOpts) {
     };
   }
 }
+
+/**
+ * Implemented this function again here from common-server.
+ *  This uses minimatch of latest version which is independent of path module
+ */
+function globMatch({
```

Comment:
can we migrate the `globMatch` from `common-server` to `common-all` instead using the latest `minimatch`?

Reply:
we can. Will migrate `globMatch` from common-server to common-all, this would also require to have minimatch of latest reason. Reason being the latest version does not have dependency on path module.